examples/freecap/freecap.css

```diff
@@ -0,0 +1,30 @@
+#captchas {
+  width: 864px;
```

Comment:
Could probably clean up the presentation here a bit since it wraps funny to me. Maybe `max-width: 500px;` or something. Right now it's somewhat hard to follow.